WebDriverAgentLib/Commands/FBCustomCommands.m

```diff
@@ -490,4 +495,50 @@ + (NSString *)timeZone
   return [localTimeZone name];
 }
 
+#if !TARGET_OS_TV // tvOS does not provide relevant APIs
++ (id<FBResponsePayload>)handleGetSimulatedLocation:(FBRouteRequest *)request
+{
+  NSError *error;
+  CLLocation *location = [XCUIDevice.sharedDevice fb_getSimulatedLocation:&error];
+  if (nil == location) {
```

Comment:
It seems like this case includes the simulated location hasn't set. Current error message is a simple server side error occurred ..... Maybe we also should the possibility the device hasn't had simulated location yet.

Reply:
it should be there: https://github.com/appium/WebDriverAgent/pull/662/files#diff-21820d38d3caab2502979871977d281a5c3032d3666fd56325ba6ebc3d8e1134R186

Reply:
Maybe the message was by XCTest framework itself